chatty-clownfish/bangazon-sprint-2
# Rh productlist

## Link to Ticket
https://github.com/chatty-clownfish/bangazon-sprint-2/issues/37

Closes # [Link]()
2/2 Ticket#13 -- still need to do 1/2

## Description of Proposed Changes
Added views.products_by_type url 
Added products_by_type method in views.py
Added products_by_type.html in templates

products_by_type method queries product and producttype tables to list individual producttype followed by list of all associated products
- 

## Steps to Test
visit url: http://localhost:8000/producttype/1 (or whatever the id is of the product type)
Product Type should be listed, followed by all associated products
*User login should not be required to view this page*

```sh
git fetch --all
git checkout rh_productlist
```

## Impacted Areas in Application
Product List by Product Type (aka category detail).

List general components of the application that this PR will affect:

urls, views.py, products_by_type.html template
* 

## Mentions @username

Tag users that need to review this code
